.travis/setup-windows.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh -ex
+
+# Git bash doesn't have rev, so here it is
+rev()
+{
+    echo "$1" | awk '{ for(i = length($0); i != 0; --i) { a = a substr($0, i, 1); } } END { print a }'
+}
+
+# Globals
+QT_BASE_URL='http://qt.mirror.constant.com/online/qtsdkrepository/windows_x86/desktop/qt5_5141/qt.qt5.5141.win64_msvc2017_64/5.14.1-0-202001240957qtbase-Windows-Windows_10-MSVC2017-Windows-Windows_10-X86_64.7z'
+QT_WIN_URL='http://qt.mirror.constant.com/online/qtsdkrepository/windows_x86/desktop/qt5_5141/qt.qt5.5141.win64_msvc2017_64/5.14.1-0-202001240957qtwinextras-Windows-Windows_10-MSVC2017-Windows-Windows_10-X86_64.7z'
+QT_DECL_URL='http://qt.mirror.constant.com/online/qtsdkrepository/windows_x86/desktop/qt5_5141/qt.qt5.5141.win64_msvc2017_64/5.14.1-0-202001240957qtdeclarative-Windows-Windows_10-MSVC2017-Windows-Windows_10-X86_64.7z'
+QT_TOOLS_URL='http://qt.mirror.constant.com/online/qtsdkrepository/windows_x86/desktop/qt5_5141/qt.qt5.5141.win64_msvc2017_64/5.14.1-0-202001240957qttools-Windows-Windows_10-MSVC2017-Windows-Windows_10-X86_64.7z'
+LLVMLIBS='https://github.com/RPCS3/llvm-mirror/releases/download/custom-build-win/llvmlibs.7z'
+GLSLANG='https://www.dropbox.com/s/6e8w6t5dxh3ad4l/glslang.7z?dl=1'
+VULKAN_SDK_URL='https://sdk.lunarg.com/sdk/download/1.1.126.0/windows/VulkanSDK-1.1.126.0-Installer.exe'
```

Comment:
Same with this Vulkan SDK version, no reason to have to insert the version number twice.